stripe/src/main/java/com/stripe/android/Stripe.java

```diff
@@ -525,6 +525,12 @@ public Token createPiiTokenSynchronous(@NonNull String personalId, String publis
                 mLoggingResponseListener);
     }
 
+    public void logEventSynchronous(@NonNull Map<String, Object> loggingMap,
+                                    @Nullable StripeApiHandler.LoggingResponseListener listener) {
+        RequestOptions options = RequestOptions.builder(mDefaultPublishableKey).build();
```

Comment:
This added public method replaces the public one in StripeApiHandler, which is needed to make logging calls from the Android Pay library.